src/integrator.jl

```diff
@@ -9,8 +9,6 @@
 
 abstract type AbstractIntegrator end
 
-NesterovDualAveraging(δ::AbstractFloat, i::AbstractIntegrator) = NesterovDualAveraging(δ, nom_step_size(i))
```

Comment:
why removing this function?

Reply:
It's moved to `adaptation.jl`. The intention is to keep all functions that are only for the purpose of user interface into the same place.

Reply:
See new comment in `adaption.jl`.